company/team/index.md

```diff
@@ -420,3 +420,11 @@ To add yourself to this page, copy the following template, paste it at the end o
 - Bio (hobbies, work experience, family, pets, etc.)
 
 -->
+
+## Chris Surdi
```

Comment:
put your entry above the template comment please

Reply:
Done!